consensus/quorum/one-node-staked-vote.go

```diff
@@ -230,7 +230,7 @@ func (v *stakedVoteWeight) JSON() string {
 		IsHarmony   bool   `json:"is-harmony-slot"`
 		Identity    string `json:"bls-public-key"`
 		VotingPower string `json:"voting-power-%"`
-		RawStake    string `json:"raw-stake,omitempty"`
+		EPoSedStake string `json:"effective-stake,omitempty"`
```

Comment:
What exactly is rawStatke? If it's the effective stake, just call it EffectiveStake here.

Reply:
right, its the stake post epos processing, the rawstake naming was misleading